### PR TITLE
Added Subject

### PR DESCRIPTION
**Pull request by naveen26887**

### CODE REVIEW

**Comment by RamuK2001** [comment]
Looks good.

File: src/main/java/com/cg/ams/entity/CourseEntity.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 
 	@Id
-    @GeneratedValue
+    //@GeneratedValue
```
Comment (phanindraduvvuri) [inline src/main/java/com/cg/ams/entity/CourseEntity.java:26]: Bro, why did you remove @GeneratedValue.

Comment (RamuK2001) [reply src/main/java/com/cg/ams/entity/CourseEntity.java:26]: Is that mandatory bro..?

Comment (naveen26887) [reply src/main/java/com/cg/ams/entity/CourseEntity.java:26]: Bro I connected the tables using ManyToOne . So at that time i changed i think. I will add it in the next commit

**Review by phanindraduvvuri — APPROVED** [review]
Looks Good.

File: src/main/java/com/cg/ams/entity/CourseEntity.java (hunk 1 above)
Comment (phanindraduvvuri) [reply src/main/java/com/cg/ams/entity/CourseEntity.java:26]: > Is that mandatory bro..?

Not mandatory, but if you add it you don't have to add ID for each entry right? 
You don't have to add it.

**Review by phanindraduvvuri — APPROVED** [review]

**Review by RamuK2001 — APPROVED** [review]

File: src/main/java/com/cg/ams/entity/CourseEntity.java (hunk 1 above)
Comment (RamuK2001) [reply src/main/java/com/cg/ams/entity/CourseEntity.java:26]: Yeah.. We can change it after merging anyway, right?
